templates/tableau-single-server-centos.template

```diff
@@ -230,47 +230,50 @@
     },
     "Mappings": {
         "AwsRegionToAMI": {
+            "AMI": {
+                "CENTOS7HVM": "CentOS-7-HVM"
```

Comment:
I'm assuming that this is a plain-vanilla CentOS AMI...

Reply:
Yes :)

Reply:
Please read comment on Ubuntu AMI... :)